lib/ex_webrtc/dtls_transport.ex

```diff
@@ -121,18 +127,25 @@ defmodule ExWebRTC.DTLSTransport do
   end
 
   @impl true
-  def handle_cast({:send_data, _data}, %{dtls_state: :connected, ice_state: ice_state} = state)
+  def handle_cast({:send_rtp, data}, %{dtls_state: :connected, ice_state: ice_state} = state)
       when ice_state in [:connected, :completed] do
-    # TODO
+    case ExLibSRTP.protect(state.out_srtp, data) do
+      {:ok, protected} -> ICEAgent.send_data(state.ice_agent, protected)
+      {:error, reason} -> Logger.error("Unable to protect RTP: #{inspect(reason)}")
+    end
+
     {:noreply, state}
   end
 
   @impl true
-  def handle_cast({:send_data, _data}, state) do
-    Logger.error(
-      "Attempted to send data when DTLS handshake was not finished or ICE Transport is unavailable"
-    )
+  def handle_cast({:send_rtp, _data}, state) do
+    Logger.warning("Attemped to send RTP before DTLS handshake has been finished")
```

Comment:
Let's explicitly state that we ignore the packet. This way we will always be sure reading docs that this behaviour doesn't have any impact on the PC.

```suggestion
    Logger.warning("Attemped to send RTP before DTLS handshake has been finished. Ignoring.")
```